ci/tests/Jenkinsfile.e2e-nightly

```diff
@@ -23,15 +33,14 @@ pipeline {
 
   stages {
     stage('Fetch') {
-      when { expression { !params.APK_URL } }
       steps { script {
         copyArtifacts(
-          projectName: "status-mobile/nightly",
-          filter: '*-x86.apk',
-          /* WARNING: This copies the latest available artifact. */
-          selector: lastWithArtifacts(),
+          projectName: "status-mobile/platforms/android-e2e",
```

Comment:
I'm not sure this path makes sense, since the `android-e2e` build is no longer part of the `combined` job.

Reply:
Ah, that's because we use another "combined" job to run nightly e2e - `ci/Jenkinsfile.e2e-nightly`

Reply:
Let me double check with CI test run:
https://ci.status.im/job/status-mobile/job/e2e-nightly/70/

Reply:
All good